container/container.go

```diff
@@ -227,3 +234,51 @@ func (a *AvroContainerWriter) AddAvroContainerWriter(p *generator.Package) {
 	p.AddFunction(a.filename(), a.name(), "WriteRecord", a.writeRecordDef())
 	p.AddFunction(a.filename(), a.name(), "Flush", a.flushDef())
 }
+
+func (a *AvroContainerWriter) AddCheckSchema(p *generator.Package) {
```

Comment:
Well this begs the question - if I run `CheckSchema` and succeeds once, can't I expect it to always succeed? Unless our schema registry goes down, gets deleted, etc, then the next time I run `CheckSchema` it might fail. How often can we expect that to happen?